docs/docs/guides/migrate/sources/auth0-guide.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Migrating Users from Auth0 to ZITADEL (Including Password Hashes)
+sidebar_label: Auth0 Migration Guide
+---
+
+## 1. Introduction
+
+**Why migrate from Auth0 to ZITADEL?**  
+This guide will walk you through the steps to migrate users from Auth0 to ZITADEL, including password hashes (with Auth0 support assistance), so users don't need to reset their passwords.
+
+**What you'll learn with this guide**
+- How to prepare your data from Auth0
+- Use of ZITADEL migration tooling
+- Performing the user import via API
+- Troubleshooting and validating the migration
+
+---
+
+## 2. Prerequisites
+
+### 2.1. Install Go
+The migration tool is written in Go. Download and install the latest version of Go from the [official Go website](https://go.dev/).
```

Comment:
Would it not make more sense to have this as a compile binary?

Not urgent but would be a good QoL improvement

Reply:
Yes, that's a good point. I can create an enhancement request for the zitadel-tools after this workshop, would that work?